klaas/scripts/train_separation_model.py

```diff
@@ -94,6 +110,7 @@ def main(configuration_path, signal_path, background_path, predictions_path, mod
     log.info('Training classifier with {} background and {} signal events'.format(
         n_protons, n_gammas
     ))
+    log.info(training_variables)
```

Comment:
Maybe add some more text to this output. Like `'Variables used for training: {}'.format(training_variables)`.If it is needed at all.